<?xml version="1.0" encoding="UTF-8"?>
<project version="4">
  <component name="DataSourceManagerImpl" format="xml" multifile-model="true">
    <data-source source="LOCAL" name="Django default" uuid="19e1fc77-ecef-443a-af91-38b916496fba">
      <driver-ref>postgresql</driver-ref>
      <synchronize>true</synchronize>
      <imported>true</imported>
      <remarks>$PROJECT_DIR$/pruebafront/settings.py</remarks>
      <jdbc-driver>org.postgresql.Driver</jdbc-driver>
      <jdbc-url>jdbc:postgresql://127.0.0.1:5432/usuarioscm</jdbc-url>
    </data-source>
    <data-source source="LOCAL" name="usuarioscm@localhost" uuid="5195cb03-df73-47f8-b576-e724e9d24231">
      <driver-ref>postgresql</driver-ref>
      <synchronize>true</synchronize>
      <jdbc-driver>org.postgresql.Driver</jdbc-driver>
      <jdbc-url>jdbc:postgresql://localhost:5432/usuarioscm</jdbc-url>
    </data-source>
    <data-source source="LOCAL" name="Django guarani3" uuid="e1c1bac0-ff79-43dd-b54e-f90ccdfff51d">
      <driver-ref>postgresql</driver-ref>
      <synchronize>true</synchronize>
      <imported>true</imported>
      <remarks>$PROJECT_DIR$/pruebafront/settings.py</remarks>
      <jdbc-driver>org.postgresql.Driver</jdbc-driver>
      <jdbc-url>jdbc:postgresql://192.168.1.145:5432/guarani</jdbc-url>
    </data-source>
  </component>
</project>